normandy/recipes/models.py

```diff
@@ -28,35 +31,54 @@ class Recipe(models.Model):
     def implementation(self):
         return ';'.join(action.implementation for action in self.actions.all())
 
+    def log_rejection(self, client=None, field=None, msg=None, op='!='):
```

Comment:
I think instead of having flexible optional arguments you should make this just take a message, and then have another method that wraps it for the special case of locale and country.

Also, `op` isn't used anywhere.